### PR TITLE
Update Dependencies (REUPLOAD)

### PR DESCRIPTION
**Pull request by CDAGaming**
Uglify-JS: 2.8.22 > 3.0.11
QUnitJS: 2.3.1 > 2.3.2
PostCSS-CLI: 3.1.1 > 4.0.0
Clean-CSS-CLI: 4.0.12 > 4.1.3
PostCSS-FlexBugs-Fixes: 2.1.1 > 3.0.0
Babel-Eslint: 7.2.2 > 7.2.3
AutoPrefixer: 6.7.7 > 7.1.1
Node-Sass: 4.5.2 > 4.5.3

### CODE REVIEW

**Comment by wolfy1339** [comment]
The Autoprefixer upgrade is a major one, relevent notes from the Changelog:

## 7.0.1
* Fix Autoprefixer for old JS runtimes.

## 7.0 “Coelestem adspicit lucem”
* Remove node.js 0.12 support.
* Use PostCSS 6.0.
* Use Browserslist 2.
* Use `caniuse-lite` instead of `caniuse-db` (by Ben Briggs).
* Use `^` for Browserslist dependencies, instead of `~`.
* Disable Grid Layout prefixes for IE by default.
* Fix `-ms-grid-column-align`.
```

**Comment by CDAGaming** [comment]
Seems Odd that on the latest commit it's referencing TWO different version of JQuery. In config.yml also, that jQuery file CDN can be adjusted to cdnjs, while I'll most likely add to this PR later today

**Comment by mdo** [comment]
This will need some reviewing and testing before we accept. Can anyone confirm all the dist files build properly and there are no errors?

**Comment by CDAGaming** [comment]
May Need to recheck these dependencies, as some may have updated once more.

**Comment by CDAGaming** [comment]
Yup, some deps need More Updating, one moment

**Comment by CDAGaming** [comment]
Local Environment and Travis Building is Successful, and Bundler Version Dependency is now also Updated

**Comment by CDAGaming** [comment]
Just Realized we are now using popper, which has a LOT of outdated dependencies, RIP 👍

**Comment by Johann-S** [comment]
It seems @bardiharborow made a test in our `deps` branch (see : https://github.com/twbs/bootstrap/commit/eb0130ad9bdaf175d11d75e76ec04d3326947893)

and Travis build failed https://travis-ci.org/twbs/bootstrap/builds/231071098

**Comment by CDAGaming** [comment]
Odd @Johann-S ,as in this PR all builds are reporting successfull, and the main errors im seeing are in relation to QUnit, ill pull the changes done in deps branch and see whats up with it

**Comment by CDAGaming** [comment]
So, looking into the changes, the deps referenced still seem to be a bit outdated. I've also noticed that 'docs/assets/js/vendor/jquery-slim.min.js' was removed, but it appears that the errors in Travis is still referencing this file. Seems at this time, that that could be a probable cause of those errors. @bardiharborow You may reference the newer commits on this PR, and try to add back 'docs/assets/js/vendor/jquery-slim.min.js', which might fix travis error

**Comment by CDAGaming** [comment]
As you can see, Updated QUnit with No issues. Most likely error cause was that deleted file most likely

**Review by Johann-S — APPROVED** [review]
LGTM 👍

File: js/tests/vendor/qunit.css [hunk 1]
```diff
@@ -433,4 +433,4 @@
 	left: -10000px;
 	width: 1000px;
 	height: 1000px;
-}
+}
```
Comment (wolfy1339) [inline js/tests/vendor/qunit.css]: Please re-add the final new-line

**Review by wolfy1339 — CHANGES_REQUESTED** [review]

File: js/tests/vendor/qunit.css (hunk 1 above)
Comment (CDAGaming) [inline js/tests/vendor/qunit.css]: It's there. was just an invisible diff, and shouldnt need to be worried about

**Pull request merged**